airflow/bin/cli.py

```diff
@@ -339,6 +339,16 @@ def run(args, dag=None):
         settings.configure_vars()
         settings.configure_orm()
 
+    # Get task attempt that will be used to separate log based on
+    # each attempt number
+    TI = TaskInstance
+    session = settings.Session()
+    ti = session.query(TI).filter(
```

Comment:
Putting it into models.py is indeed better. Just curious what could go wrong if I directly access db here?